cache/in-memory/src/lib.rs

```diff
@@ -1170,6 +1211,13 @@ mod tests {
             }
             _ => panic!("{:?}", channel),
         }
+
+        match *thread {
```

Comment:
This shouldn't have to be dereferenced?

Reply:
Negative, you get a type mismatch of ``expected struct `Arc`, found enum `GuildChannel``